<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KRGYeniKargoEkle</class>
 <widget class="QDialog" name="KRGYeniKargoEkle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yeni Kargo Ekle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLabel" name="lblYeniKargo">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Yeni Kargo</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9" stretch="2,5,3">
         <item>
          <widget class="QGroupBox" name="groupBoxOdemeTuru">
           <property name="title">
            <string>Ödeme Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
              <item>
               <widget class="QRadioButton" name="radiobtnAliciOdemeli">
                <property name="text">
                 <string>Alıcı Ödemeli</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radiobtnGonderiOdemeli">
                <property name="text">
                 <string>Gönderi Ödemeli</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxDesi">
           <property name="title">
            <string>Desi Hesaplama</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3">
                <item>
                 <widget class="QLabel" name="lblDesiEn">
                  <property name="text">
                   <string>En</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dspinboxDesiEn"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3">
                <item>
                 <widget class="QLabel" name="lblDesiBoy">
                  <property name="text">
                   <string>Boy</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dspinboxDesiBoy"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,3">
                <item>
                 <widget class="QLabel" name="lblDesiYukseklik">
                  <property name="text">
                   <string>Yükseklik</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dspinboxDesiYukseklik"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,3">
                <item>
                 <widget class="QLabel" name="lblDesiAgirlik">
                  <property name="text">
                   <string>Ağırlık</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dspinboxDesiAgirlik"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,1">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="lblDesiSonuctanOncekiLabel">
                  <property name="text">
                   <string>Desi:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblDesiSonucSayi">
                  <property name="text">
                   <string>20</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxDetaylar">
           <property name="title">
            <string>Detaylar</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,3">
                <item>
                 <widget class="QLabel" name="lblGondericiSube">
                  <property name="text">
                   <string>Gönderici Şube</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboboxGondericiSube"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="2,3">
                <item>
                 <widget class="QLabel" name="lblAliciSube">
                  <property name="text">
                   <string>Alıcı Şube</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboboxAliciSube"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="2,3">
                <item>
                 <widget class="QLabel" name="lblTarih">
                  <property name="text">
                   <string>Tarih</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="dateEditTarih"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBoxAliciDetay">
           <property name="title">
            <string>Alıcı</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="2,3">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Alıcı Adı</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_5"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Alıcı Adresi</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="2,3">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Telefon Numarası:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_7"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="2,3">
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>Email:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_8"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxGondericiDetay">
           <property name="title">
            <string>Gönderici</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="2,3">
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>Gönderici Adı</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_9"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>Gönderici Adresi</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="plainTextEdit_3"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="2,3">
                <item>
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>Telefon Numarası:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_10"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="2,3">
                <item>
                 <widget class="QLabel" name="label_21">
                  <property name="text">
                   <string>Email:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_11"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,6,1,1">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Tutar:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>20</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>İptal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Kaydet</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
